src/Template/Layout/default.ctp

```diff
@@ -6,13 +6,12 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
     <!-- Base CSS -->
-    <?= $this->Html->css('CrudView.bootstrap.min');?>
-    <?= $this->Html->css('CrudView.bootflat.min');?>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css">
     <?= $this->Html->css('CrudView.local');?>
 
     <!-- Base JS -->
     <?= $this->Html->script('CrudView.jquery.min');?>
-    <?= $this->Html->script('CrudView.bootstrap.min');?>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>
 
     <!-- Moment.js -->
     <?= $this->Html->script('CrudView./contrib/momentjs/js/moment-with-langs.min');?>
```

Comment:
On a related-note to my above comments, that leaves the scope of this pr: [this kind of sprawling-jungle of dependencies in the layout](https://github.com/FriendsOfCake/crud-view/blob/josegonzalez-patch-1/src/Template/Layout/default.ctp#L4-L33) is something I think should be avoided.

aim for:

```
<head>
    <?= $this->Html->charset(); ?>
    <title><?= $this->get('title');?></title>
    <?= $this->fetch('meta'); ?>
    <?= $this->fetch('css'); ?>
    <?= $this->fetch('head-script'); ?>
</head>
```

So that the template file is clean and extensible. Also don't put any js in the head that isn't absolutely necessary (read: basically never more than modernizr or similar).